app/admin/golem/jobs/page.tsx

```
@@ -0,0 +1,420 @@
+          setSelectedJob(job);
+          if (job.status === 'new') {
+            updateJobStatus(job.id, 'viewed');
+          }
+        }}
+        className={`w-full text-left p-4 rounded-lg border transition-all ${
+          isSelected
+            ? 'bg-white/10 border-white/30'
+            : 'bg-white/5 border-white/10 hover:bg-white/[0.07]'
+        }`}
+      >
+        <div className="flex items-start justify-between gap-2 mb-2">
```

Biome (2.3.13)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)